flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/slot/TaskSlotTableBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.taskexecutor.slot;
+
+import org.apache.flink.api.common.time.Time;
+import org.apache.flink.runtime.clusterframework.types.AllocationID;
+import org.apache.flink.runtime.clusterframework.types.ResourceProfile;
+import org.apache.flink.runtime.memory.MemoryManager;
+import org.apache.flink.runtime.taskexecutor.TaskManagerServices;
+import org.apache.flink.runtime.testingUtils.TestingUtils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/** Builder for {@link TaskSlotTable}. */
+public class TaskSlotTableBuilder {
+	private static final long DEFAULT_SLOT_TIMEOUT = 10000L;
+
+	private static final ResourceProfile DEFAULT_RESOURCE_PROFILE =
```

Comment:
why are we now using a different default profile? Previously, it was ResourceProfile.ANY, which also seems to be a safer default.

Reply:
I want to avoid creation of slot `MemoryManager` with infinite memory. It is just a dummy memory size anyways by default.